inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -351,6 +351,10 @@ void GNAGraphCompiler::PowerPrimitive(InferenceEngine::CNNLayerPtr layer) {
     uint32_t num_rows_out = num_rows_in;
     uint32_t num_padding = ALIGN(num_rows_in, 8) - num_rows_in;
 
+    if (num_columns_in > 8) {
+        THROW_GNA_EXCEPTION << "GNA colomns_in should be less than 8 and now colomns_in = " << num_columns_in;
```

Comment:
Several issues in the error message:
1. Should be "columns_in", not "colomns_in".
2. No need to mention GNA - it's already mentioned by the macro.
3. It claims that "it should be less than 8", but it's wrong. It should be not greater than 8, not less.
4. It's two low level. Why not explain what's wrong from the user's perspective?

Can we add a test case for this error message?
And please rebase.

Reply:
What about note 4?